<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPropertiesDialog</class>
 <widget class="QDialog" name="TaskPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Summary:</string>
     </property>
     <property name="buddy">
      <cstring>summaryEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="summaryEdit">
     <property name="toolTip">
      <string>Enter the task summary</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>descriptionEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="toolTip">
      <string>Enter the task description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Creation Time:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="creationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>&amp;Priority:</string>
     </property>
     <property name="buddy">
      <cstring>priorityCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="priorityCombo">
     <property name="toolTip">
      <string>Select the task priority</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="projectCheckBox">
     <property name="toolTip">
      <string>Select to indicate a task related to the current project</string>
     </property>
     <property name="text">
      <string>Project &amp;Task</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="completedCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select to mark this task as completed</string>
     </property>
     <property name="text">
      <string>T&amp;ask completed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="text">
      <string>Filename:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="filenameEdit">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel6">
     <property name="text">
      <string>Line:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="linenoEdit">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>summaryEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>priorityCombo</tabstop>
  <tabstop>projectCheckBox</tabstop>
  <tabstop>completedCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
